nautobot/core/templatetags/helpers.py

```diff
@@ -238,6 +238,25 @@ def viewname(model, action):
     return lookup.get_route_for_model(model, action)
 
 
+@library.filter()
+@register.filter()
+def api_viewname(model, action):
```

Comment:
No longer used now - do you think it's still needed/desired?

Reply:
Personally, I think it is redundant for the time being and can be removed.

Reply:
Ok, makes sense. Removed.